RZUtils/Components/RZAbout/RZAbout.m

```diff
@@ -0,0 +1,215 @@
+}
+
+- (void)mailComposeController:(MFMailComposeViewController *)controller didFinishWithResult:(MFMailComposeResult)result error:(NSError *)error
+{
+    if (self.feedbackBlock) {
+        self.feedbackBlock(result, error);
+    }
+
+    [controller dismissViewControllerAnimated:YES completion:nil];
+}
+
+#pragma mark - Sharing
+
+- (void)presentShareAppInController:(UIViewController *)viewController shareText:(NSString *)shareText shareURL:(NSURL *)shareURL
+{
+    [self presentShareAppInController:viewController shareText:shareText shareURL:shareURL anchorViewForPad:nil anchorFrame:CGRectNull];
+}
+
+- (void)presentShareAppInController:(UIViewController *)viewController shareText:(NSString *)shareText shareURL:(NSURL *)shareURL anchorViewForPad:(UIView *)anchorView anchorFrame:(CGRect)frame
+{
+    NSParameterAssert(viewController);
+    NSParameterAssert(shareText);
+    NSParameterAssert(shareURL);
+
+    UIActivityViewController *activityVC = [[UIActivityViewController alloc] initWithActivityItems:@[shareText, shareURL] applicationActivities:nil];
+
+    if (anchorView) {
+        activityVC.popoverPresentationController.sourceView = anchorView;
+        activityVC.popoverPresentationController.sourceRect = frame;
+    }
+
+    [viewController presentViewController:activityVC animated:YES completion:nil];
+}
+
+#pragma mark - Built by RZ
+
++ (UIView *)RZLogoViewConstrainedToWidth:(CGFloat)maxWidth
+{
+    CGFloat viewHeight = 140.0f;
+    CGFloat imageHeight = 40.0f;
+    CGFloat labelHeight = 30.0f;
+
+    UIView *logoView = [[UIView alloc] initWithFrame:CGRectMake(0.0f, 0.0f, maxWidth, viewHeight)];
+    logoView.backgroundColor = [UIColor clearColor];
+
+    UIImageView *logoImageView = [[UIImageView alloc] initWithFrame:CGRectMake(0.0f, viewHeight - labelHeight - imageHeight, logoView.frame.size.width, imageHeight)];
+    logoImageView.backgroundColor = [UIColor clearColor];
+    logoImageView.contentMode = UIViewContentModeCenter;
+
+    NSURL *imageURL = [[NSBundle bundleForClass:[self class]] URLForResource:@"logo-built-by-RZ" withExtension:@"png"];
+    if (imageURL != nil) {
+        logoImageView.image = [UIImage imageWithContentsOfFile:[imageURL path]];
+    }
+
+    [logoView addSubview:logoImageView];
+
+    UILabel *buildLabel = [[UILabel alloc] initWithFrame:CGRectMake(0.0f, viewHeight - labelHeight, logoView.frame.size.width, labelHeight)];
+    buildLabel.backgroundColor = [UIColor clearColor];
+    buildLabel.textAlignment = NSTextAlignmentCenter;
+    buildLabel.textColor = [UIColor colorWithRed:0.5 green:0.5 blue:0.51 alpha:1];
+    buildLabel.font =  [UIFont systemFontOfSize:15.0f];
+    buildLabel.text = [NSString stringWithFormat:@"%@ (%@)", [[self class] appVersion], [[self class] appBuild]];
+    [logoView addSubview:buildLabel];
+
+    return logoView;
+}
+
+#pragma mark - App Info
+
++ (NSString *)systemVersion
+{
+    return [UIDevice currentDevice].systemVersion ?: @"Unknown";
```

Comment:
#? why use hardcoded English `'Unknown'` here (and next 3 methods) when you use `NSLocalizedString()` macro in `appInfoText()`?